biome.json

```diff
@@ -1,9 +1,12 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
   "linter": {
     "enabled": true,
     "rules": {
       "recommended": true,
+      "a11y": {
+        "useSemanticElements": "off"
```

Comment:
can we add explanation in the PR why those changes were needed? have some defaults changed?

Reply:
Yes, good suggestion. Added as main comment to PR, will include in merge commit message